src/main/java/org/example/bookshop/BookShopApplication.java

```diff
@@ -0,0 +1,20 @@
+package org.example.bookshop;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class BookShopApplication {
+    public static void main(String[] args) {
+        SpringApplication.run(BookShopApplication.class, args);
+    }
+
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> {
+            System.out.println("123");
```

Comment:
implement main according to the task description

Reply:
if I'm not mistake we don't have any requirement about main class in description?

Reply:
Checked, you rigth
maybe there are some requirements in video. You need to call methods from your service to check how they work